.github/workflows/build-ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Build CI
```

Comment:
1) Add badge to readme.
2) Do we want the image build here or in the core repo?  I thought that we were putting all of this stuff in the core repo.  If here, I assume "this stuff" should get pulled from core.

Reply:
I *think* it better lives here, alongside the sample data. That way the image and the data can be versioned together in a single step, as opposed to separately.

Reply:
I agree.  We just need a PR to remove the stuff from core.